### ISSUE TITLE
msgpack beta fails to install

### ISSUE THREAD

**Issue by shivammathur** [mlocati/powershell-phpmanager#67]
This fails to find DLL for `PHP 7.4` while it exists in [`2.1.0beta1`](https://pecl.php.net/package/msgpack/2.1.0beta1/windows)

```ps1
Install-PhpExtension -Extension msgpack -MinimumStability beta -MaximumStability beta -Path C:\tools\php
```

Test Workflow - https://github.com/shivammathur/test-setup-php/runs/1321894694?check_suite_focus=true#step:4:1

**Comment by mlocati** [comment]
Thanks for the report: fixed by #68
I'll publish a new version asap

**Comment by mlocati** [comment]
I've just released a new version (1.24.1), which includes #68

**Comment by shivammathur** [comment]
@mlocati Thanks for fixing it.
